content/en/apps/guides/messaging/gateways/gateway/troubleshooting.md

```diff
@@ -50,8 +50,36 @@ grep api/sms medic_medic_medic-api4.log
 ```
 If the above steps don't yield the problem, read the [Obtaining Logs]({{< ref "apps/guides/debugging/obtaining-logs#android-logs" >}}) page, note the make/model/android version their gateway handset is and escalate with those details to the PM/project techlead.
 
+**Verifying the SMS Gateway via a Test Message**
+
+We can verify whether the SMS Gateway app is responding by sending a test message via the web app. For example, let us try this in a deployed project.
+1. Login via the web app.
+
+![image](https://user-images.githubusercontent.com/3299006/175019916-2319fd9d-fbd3-4dea-9a2e-580b5aca559d.png)
+
+2. Goto App Management.
+
+![image](https://user-images.githubusercontent.com/3299006/175015115-d3cc84c9-937b-453f-b2b0-97e7122cb211.png)
+
+3. Click on SMS option, then click the Test Message tab.
+
+![image](https://user-images.githubusercontent.com/3299006/175015577-e546c897-555c-46c7-b77f-131f12baa669.png)
+
+4. The Gateway replies with the status message on the phone number we input in the "From phone number" field. So please input the test message and a valid phone number where you expect to view the status message and press the "Send message" button. We should see the Report Submitted message if the SMS was enqueued to be sent.
```

Comment:
I assume the `message body` doesn't matter at all, right?

Also, on this screenshot there is no country code prefix but in the next screenshots I see country code and the + symbol. Maybe you could clarify that. 

Finally, i would probably phrase it something like this:

Enter a `Message body`, this is the message that will get sent to the gateway. Then enter a valid phone number (explain what "valid" means, per my comment above) into `From phone number`, this is the phone number that gateway will respond to.  Then press the `Send message` button.  If it was successfully sent, you should see a `Report Submitted` message.

Reply:
Thanks. I have rephrased it.